algorithms/sorting/Insertion_Sort_Part2.java

```diff
@@ -0,0 +1,46 @@
+import java.io.*;
```

Comment:
Nice job!
Please, change the name to lowercase to keep the standard of other solutions.
A nice name would be `insertion_sort_part_2.java` to keep standard with the other language versions that already is in this repo.
Please, fix the indentation and remove whitelines too.